src/components/Navbar.js

```diff
@@ -125,8 +144,8 @@ const Navbar = ({ toggleDarkMode, mode }) => {
                 <Button color="inherit" component={Link} to="/profile">
                   Profile
                 </Button>
-                <Button color="inherit" component={Link} to="/wishlist">
-                  Wishlist
+                <Button component={Link} to="/wishlist">
+                  <FavoriteIcon />
```

Comment:
done!

Reply:
Looks good! 

Do we intentionally want only the wishlist to be an icon button in the hamburger menu button for mobile? I think it would be better to keep it consistent, either have the whole hamburger menu be words or whole hamburger menu be icons. But I'm not super-design-minded, so I'm happy to proceed however you prefer!

Reply:
@akosasante thanks for the advice, Akos!! I totally agree with you that it´s better to stick to the same pattern, I am a desastrous designer haha so I appreciate any your idea and suggestion!❤️